src/sora-v3.md

```diff
@@ -0,0 +1,116 @@
+
+::: info
+The Chameleon Pool model is no longer active.
+:::
+
+Keep in mind that increasing the amount of goods and services in an economy is crucial, by definition, to expand the GDP. Therefore, any viable economic system needs to keep in mind ΔMV=ΔPY and understand that **you need an expansion of purchasing power for production to create more goods/services**. SORA v3 tokenomics follows the [General Quantity Theory of Money (Disaggregated Quantity Theory of Money)](https://cyberleninka.ru/article/n/banks-and-economic-growth-the-general-theory-in-a-basic-disequilibrium-model-with-five-rationing-regimes) so our economy can expand sustainably and reliably, which will enable us to accomplish our goal of creating a new economic order for the world that enables collaboration instead of exploitation and competition. _We will all own XOR, and we will be happy._
+
+### Second Iteration
+
+![](/.gitbook/assets/sora-v3-3.png)
+
+Given the current state of the world, like it or not, even cryptoland is dollarized. That means SORA must also provide access to tokens at par with the dollar if we want to increase market share. Fortunately, _Kensetsu_ (SORA’s version of MakerDAO) is a natural way for us to provide this access to our users.
+
+However, given the low liquidity for Kensetsu assets up until now, especially KUSD, there haven’t been any good options for SORA users to access stable USD values. So, to build the SORA economy into one that is useful in daily life, we, as an ecosystem, should work to maintain the peg of at least KUSD so that there is reliable liquidity for a token with dollar value, which is the current world standard (until XOR takes over).
+
+#### BUIDLing NEO
+
+Creating financial software demands the highest quality code and repeatable processes; otherwise, it cannot be reliably used. Teams that BUIDL in the SORA ecosystem are expected to deliver the best results and have consistently proven themselves. However, professional teams cannot be expected to work for free, nor should they. Because we are still in the BUIDL phase of creating the SORA economic system, we should pay teams that work in the SORA ecosystem in a unit of account that is readily understandable, namely w.r.t. dollars (USD, also known as _cuck bucks_).
+
+So, as part of the improvements to SORA on our journey to v3, KUSD is now proposed to be used to pay builders. To create the necessary KUSD and have it maintain its peg, it is important to use only hard collateral (not soft cats😿). For now, this means using TBCD as collateral. Even though we will need to mint **a lot of TBCD (one hundred billion to be exact)** to lock in Kensetsu vaults to mint KUSD, these TBCD won’t actually be converted to XOR, so it won’t affect the XOR token supply. It is just important to have secure collateral in Kensetsu vaults to build up confidence in the system.
```

Comment:
_"For now, this means using TBCD as collateral. Even though we will need to mint **a lot of TBCD (one hundred billion to be exact)** to lock in Kensetsu vaults [...]"_

It is mentioned in the SORA V3 Tokenomics section that TBCD minting has been stopped. Those sentences sound in contradiction with what is exposed in the aforementioned section.

Reply:
Edited